src/util/__tests__/validateRoam.ts

```diff
@@ -70,10 +70,10 @@ test('it returns false if there are any missing properties', () => {
   type requiredKeyTypes = 'create-email' | 'uid' | 'create-time' | 'string'
   const requiredKeys = ['create-email', 'uid', 'create-time', 'string']
   requiredKeys.forEach(key => {
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     const invalidRoamString = JSON.stringify([
       {
         ...testData[0],
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
         children: testData[0].children.map(({ [key as requiredKeyTypes]: requiredKey, ...rest }) => ({ ...rest })),
```

Comment:
@raineorshine 
here requiredKey is unused var. if remove this it will throw error.

Reply:
Does replacing `requiredKey` with `_` work? Sometimes the underscore is used as a blank when a variable is not used.

Reply:
let me check if it works. otherwise will add the command to ignore.

Reply:
> Does replacing `requiredKey` with `_` work? Sometimes the underscore is used as a blank when a variable is not used.

its not working. by replacing with _. got same error.

Reply:
Okay, good to know. In that case I guess we need to disable the linter and leave a comment that explains that we are just using destructuring to get the `rest`.